R/options.R

```diff
@@ -253,7 +263,9 @@ test_options <- function(title, admin_password,
        allow_url_rewrite = allow_url_rewrite,
        js_opt = list(advance_delay = advance_delay),
        additional_scripts = additional_scripts,
-       logo_position = logo_position)
+       logo_position = logo_position,
+       on_session_ended_fun = on_session_ended_fun,
+       on_stop_fun = on_stop_fun)
```

Comment:
I think `on_start_fun` is missing here?

Reply:
Interesting, this didn't raise an error, and maybe my log was too similar to the `cat` message that I didn't notice it trickle. But yes, it should have been there.

Reply:
OK, thanks! Maybe worth testing just one more time to make sure that the log happens as you expect?